packages/strapi-plugin-i18n/config/functions/migrations/field/index.js

```diff
@@ -16,16 +16,16 @@ const after = async ({ model, definition, previousDefinition, ORM }) => {
   const localizedAttributes = ctService.getLocalizedAttributes(definition);
   const prevLocalizedAttributes = ctService.getLocalizedAttributes(previousDefinition);
   const attributesDisabled = difference(prevLocalizedAttributes, localizedAttributes);
-  const attributesToMigrate = intersection(Object.keys(definition.attributes), attributesDisabled);
+  const attrsToMigrate = intersection(keys(definition.attributes), attributesDisabled);
```

Comment:
we usually try to avoid abbreviations in the code any specific reason for this change ?